cms/templates/js/self-paced-due-date-editor.underscore

```diff
@@ -0,0 +1,16 @@
+<ul class="list-fields list-input datepair date-setter">
+    <li class="field field-text field-due-in">
+        <label for="due_in"><%- gettext('Due in:') %></label>
+        <input type="number" id="due_in" name="due_in" value=""
+            placeholder="" autocomplete="off"/> weeks
+    </li>
+</ul>
+
+<ul class="list-actions">
+    <li class="action-item">
+        <a href="#" data-tooltip="<%- gettext('Clear Grading Due Date') %>" class="clear-date action-button action-clear">
+            <span class="icon fa fa-undo" aria-hidden="true"></span>
+            <span class="sr"><%- gettext('Clear Grading Due Date') %></span>
```

Comment:
Might be more of a Hilary/Kaitlin question, but if you can set due dates on ungraded content, feels weird that this explicitly calls out "Grading"